apps/web/src/components/SearchRequestFilters/SearchRequestFilters.tsx

```diff
@@ -172,7 +130,7 @@ const ApplicantFilters = ({
       <div data-h2-flex-item="base(1of1) p-tablet(1of2)">
         <div>
           <FilterBlock
-            title={intl.formatMessage(messages.community)}
+            title={intl.formatMessage(adminMessages.community)}
```

Comment:
Should we add a new community message to talentRequestsMessages as well? Then the component will only rely on one file. If you decide to do this then we should add stream (from processMessages.stream) as well.

Reply:
It's a bit tidier, nice!
[copy messages](https://github.com/GCTC-NTGC/gc-digital-talent/pull/11653/commits/78728319dd85a7a3cef18f87095b97c6fe6b70e9)